source/appModules/eclipse.py

```diff
@@ -15,6 +15,9 @@
 import keyboardHandler
 from scriptHandler import script
 
+# Translators: The name of a category of NVDA commands.
+categorySCRCAT_ECLIPSE = _("Eclipse")

```

Comment:
Sorry for the wrong suggestion from my side. Here is the correct one so that constants for category names all look the same.
```suggestion
SCRCAT_ECLIPSE = _("Eclipse")
```